management-app/backend/app_management/app_manager.py

```diff
@@ -0,0 +1,100 @@
+from conglobo_environment import CongloboEnvironment
+from app_management.app_manager_config import AppManagerConfig
+from app_management.app import App
+
+from kubernetes import client, config
+
+
+class AppAlreadyExists(Exception):
+    """Exception thrown when App already exists"""
+
+
+class AppDoesNotExist(Exception):
+    """Exception thrown when App does not exist"""
+
+
+class AppManager:
```

Comment:
This class is what can `add_app`s and `delete_app`s.

There is a helper class called `AppManagerConfig`, which generally stores config-related information for this class. I'm not sure if this 2-class approach is any good: see `manage_app_demo.py` to see some of the awkwardness.